### DISCUSSION TITLE
First/Top Task not fully visible in Checklist

### DISCUSSION THREAD

**Discussion by xthrk** [standardnotes/forum#2892] [General]
In the browser (Firefox 106.0.5, Windows 10) the first task in the checklist is not displayed correctly:
![grafik](https://user-images.githubusercontent.com/116081825/200165043-39436ddd-77b4-41a8-881e-f8f276ff39aa.png)

In the desktop app (v3.23.273, Windows 10) the checklist is displayed completely:
![grafik](https://user-images.githubusercontent.com/116081825/200165331-731fba8a-b8ca-4fe2-b15b-05745f9b46a6.png)

**Comment by effieeee** [comment]
Hi @crophit, I took note of a similar report in the help inbox, and I'll be reporting this to the team! As an additional question though, after you add or remove a task from the list, does the line spacing issue appear resolved, or does it recur after relaunching the app?

**Comment by xthrk** [comment]
Thank you. I tested this and found that as soon as I add a character in the text field for the new task or in the title, the line spacing problem immediately disappears. As soon as I exit the checklist note without restarting the app and go back to the checklist note, the problem is present again. And the problem is also present again as soon as I restart the app.

**Comment by microhertz** [comment]
Hello. I stumbled upon this issue recently and found this discussion from issues #2430 and #2266. As a work around I used the [Stylus extension](https://addons.mozilla.org/en-US/firefox/addon/styl-us/) in Firefox to create a rule for the tasklist's iframe URL `https://app.standardnotes.com/components/assets/org.standardnotes.simple-task-editor/dist/index.html`.

And added this CSS rule:
```
.task textarea.task-input-textarea {
    min-height: 1.25em;
}
```
This appears to fix the issue for me, so maybe it will work for others.
